content/docs/development/dev-dependencies.md

```diff
@@ -4,7 +4,7 @@ description: ""
 date: 2020-09-30
 draft: false
 toc: true
-weight: 100
+weight: 3100
```

Comment:
I think that the default sort is alphabetical when files have the same weight.
`dev-integration-tests.md` and `dev-releases.md` both have a weight of 3100 set. I'd suggest being more explicit about the weights, even if you want alphabetical sorting, I'd set the weights that way. 
(lower weight gets higher precedence)